code401Challenges/src/main/java/code401Challenges/hashtable/Hashtable.java

```diff
@@ -0,0 +1,79 @@
+package code401Challenges.hashtable;
+
+
+
+public class Hashtable<V> {
+
+    // Instance variables
+    // Set to generic for different possible key:value pairs
+
+    private Node<String, V> [] map;
+    private final int initialSize = 1024;
+    private final int primeValue = 4096;
+
+    //Constructor
+
+    public Hashtable() {
+
+        this.map = new Node[initialSize];
```

Comment:
The type of Node is not specified, make sure to use something like Node<String, V> !